packages/frontend/src/pages/QuantifyPeriodReceiver/components/QuantifyMultipleDialog.tsx

```diff
@@ -0,0 +1,63 @@
+import ScrollableDialog from '@/components/ScrollableDialog';
+import QuantifyMultipleButton from '@/pages/QuantifyPeriodReceiver/components/QuantifyMultipleButton';
+import QuantifySlider from '@/pages/QuantifyPeriodReceiver/components/QuantifySlider';
+import { faCalculator, faTimes } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { PraiseDto } from 'api/dist/praise/types';
+import { useState } from 'react';
+
+interface QuantifyMultipleDialogProps {
+  open: boolean;
+  onClose(): void;
+  selectedPraises: PraiseDto[];
+  allowedValues: number[];
+  onSetScore(newScore: number, selectedPraises: PraiseDto[]);
+}
+
+const QuantifyMultipleDialog = ({
+  open = false,
+  onClose,
+  selectedPraises,
+  allowedValues,
+  onSetScore,
+}: QuantifyMultipleDialogProps): JSX.Element | null => {
+  const [score, setScore] = useState<number>(0);
+
+  return (
+    <ScrollableDialog open={open} onClose={onClose}>
+      <div className="w-full h-full">
+        <div className="flex justify-end p-6">
+          <button className="praise-button-round" onClick={onClose}>
+            <FontAwesomeIcon icon={faTimes} size="1x" />
+          </button>
+        </div>
+        <div className="px-20 space-y-6">
+          <div className="flex justify-center">
+            <FontAwesomeIcon icon={faCalculator} size="2x" />
+          </div>
+          <h2 className="text-center">
+            Quantify selected ({selectedPraises.length}) praises
```

Comment:
Should we list the id's of the praise items that will be affected, like in the DismissDialog? (I don't feel strongly I just think they should be consistent).
![Screenshot_2022-06-26_12-52-23](https://user-images.githubusercontent.com/159270/175825643-a8271113-e038-49c6-b3cd-2481dc181fa3.png)